api4/api.go

```diff
@@ -236,6 +240,7 @@ func Init(a *app.App, root *mux.Router) *API {
 	api.InitScheme()
 	api.InitImage()
 	api.InitTermsOfService()
+	api.initLTI()
```

Comment:
Would this be `InitLTI` instead of `initLTI`?

Reply:
Done.